docs/source/extensions/metrics.rst

```diff
@@ -258,6 +258,50 @@ In practise this means that:
     val = metric(pred, target) # this value can be backpropagated
     val = metric.compute() # this value cannot be backpropagated
 
+
+Metric API
+----------
+
+.. autoclass:: pytorch_lightning.metrics.Metric
+    :noindex:
+
+Internal implementation details
+-------------------------------
+
+This section briefly describe how metrics work internally. We encourage looking at the source code for more info.
+Internally, Lightning wraps the user defined ``update()`` and ``compute()`` method. We do this to among other to
```

Comment:
"We do this to among other to"

Mind rephrasing?

Reply:
Done :]